GenderPayGap.WebUI/Helpers/IntegrationChecksHelper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace GenderPayGap.WebUI.Helpers
+{
+    public static class IntegrationChecksHelper
+    {
+        public static Boolean SumNotEqualTo100(decimal a, decimal b)
+        {
+            return a + b != 100;
+        }
+
+        public static Boolean FigureGreaterThan100OrLessThan(decimal? a, decimal minValue)
+        {
+            return a > 100 || a < minValue;
```

Comment:
what does this do if `a` doesn't have a value?

Reply:
it returns false